bin/q

```diff
@@ -418,12 +424,12 @@ class Sql(object):
             qtable_name] = effective_table_name
 
     def get_effective_sql(self,original_names=False):
-        if len(filter(lambda x: x is None, self.qtable_name_effective_table_names)) != 0:
+        if len(list(filter(lambda x: x is None, self.qtable_name_effective_table_names))) != 0:
```

Comment:
list comprehension is easier to understand.

Reply:
true. decided not to change it now, will do later on.